proper_paranthetics.py

```diff
@@ -0,0 +1,27 @@
+def fn(s):
+    """
```

Comment:
There are some linting errors here. Please have all the linting errors dealt with before you submit.